src/System.Memory/tests/TestHelpers.cs

```diff
@@ -462,5 +520,35 @@ public static Memory<T> DangerousCreateMemory<T>(object obj, int offset, int len
             { new string[] { "1", null, null }, new string[] { null, null, "2" }, -1},
             { new string[] { null, null, null }, new string[] { null, null }, 1},
         };
+
+        public static TheoryData<string[], string[], int> LastIndexOfAnyNullSequenceData => new TheoryData<string[], string[], int>()
+        {
+            { new string[] { "1", null, "2" }, new string[] { "1", null, "2" }, 2},
+            { new string[] { "1", null, "2" }, new string[] { null }, 1},
+            { new string[] { "1", null, "2" }, null, 0},
+            { new string[] { "1", null, "2" }, new string[] { "3", null }, 1},
+            { new string[] { null, null, "2" }, new string[] { "3", null }, 1},
+
+            { new string[] { "1", "3", "2" }, new string[] { "1", null, "2" }, 2},
+            { new string[] { "1", "3", "2" }, new string[] { null }, -1},
+            { new string[] { "1", "3", "2" }, null, 0},
+
+            { null, new string[] { "1", null, "2" }, -1},
+
+            { new string[] { "1", null, null }, new string[] { null, null, "2" }, 2},
+            { new string[] { null, null, null }, new string[] { null, null }, 2},
+        };
+
+        public static TheoryData<string[], string[], int> LastIndexOfAnyNullData => new TheoryData<string[], string[], int>()
+        {
+            { new string[] { "1", null, "2" }, new string[] { "1", null }, 1},
+            { new string[] { "1", null, "2" }, new string[] { null, null }, 1},
+            { new string[] { "1", null, "2" }, new string[] { "1", "2" }, 2},
+            { new string[] { null, null, "2" }, new string[] { null, "1" }, 1},
+            { new string[] { "1", null, "2" }, new string[] { "3", "4" }, -1},
+            { new string[] { null, null, "2" }, new string[] { null, "1" }, 1},
+            { new string[] { "1", "3", "2" }, new string[] { null, "1" }, 0},
+            { null, new string[] { null, "1" }, -1},
+        };
```

Comment:
Why do we need both LastIndexOfAnyNullSequenceData and LastIndexOfAnyNullData ?  We can't just have one test that takes the input data and calls both overloads?  Also, there are overloads that take three individual elements; where are those being tested?